flink-table/flink-table-planner/pom.xml

```diff
@@ -268,8 +279,7 @@ under the License.
 									<include>net.hydromatic:*</include>
 									<include>com.esri.geometry:*</include>
 
-									<!-- flink-table dependencies -->
-									<include>org.apache.flink:flink-table-common</include>
```

Comment:
Why don't we include `table-common` anymore?

Reply:
Looks like flink-table-planner only bundles non-Flink dependencies, and flink-table-dist bundles the various Flink dependencies.

Reply:
Yes, `flink-table-common` is pulled in when people use a Maven projects. Otherwise `flink-table-uber` contains everything needed.